Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -605,6 +605,11 @@ def audit_desc
   def audit_homepage
     homepage = formula.homepage
 
+    unless homepage && !homepage.empty?
```

Comment:
Sure! That's what I had originally - I changed it to match the more-or-less identical logic in the `desc` audit.